apis/s3/src/main/java/org/jclouds/s3/options/ListBucketOptions.java

```diff
@@ -37,6 +37,15 @@
 public class ListBucketOptions extends BaseHttpRequestOptions implements Cloneable {
    public static final ListBucketOptions NONE = new ListBucketOptions();
 
+   public ListBucketOptions withListType(String listType) {
+      queryParameters.put("list-type", checkNotNull(listType, "listType"));
+      return this;
+   }
+
+   public String getListType() {
```

Comment:
Similarly, should this be `Integer` (matching `getMaxResults`)?

Reply:
Again here the value returned would be either 2 or null.